var/spack/repos/builtin/packages/sst-elements/package.py

```diff
@@ -69,35 +95,52 @@ def autoreconf(self, spec, prefix):
         bash('autogen.sh')
 
     def configure_args(self):
+        spec = self.spec
         args = []
-        if '+pdes_mpi' in self.spec["sst-core"]:
-            env['CC'] = self.spec['mpi'].mpicc
-            env['CXX'] = self.spec['mpi'].mpicxx
-            env['F77'] = self.spec['mpi'].mpif77
-            env['FC'] = self.spec['mpi'].mpifc
 
-        if "+pin" in self.spec:
-            args.append("--with-pin=%s" % self.spec["intel-pin"].prefix)
+        if '+pdes_mpi' in spec["sst-core"]:
+            env['CC'] = spec['mpi'].mpicc
+            env['CXX'] = spec['mpi'].mpicxx
+            env['F77'] = spec['mpi'].mpif77
+            env['FC'] = spec['mpi'].mpifc
+
+        if "+pin" in spec:
+            args.append("--with-pin=%s" % spec["intel-pin"].prefix)
+
+        if "+dramsim2" in spec or "+hybridsim" in spec:
+            args.append("--with-dramsim=%s" % spec["dramsim2"].prefix)
```

Comment:
Was it intentional to replace 

```
args.append("--with-hybridsim=%s" % self.spec["hybridsim"].prefix)
```

with this? I still see a `hybridsim` dependency.

Reply:
This section's diff is a bit confusing. Most of the change is the result of replacing `self.spec` with a `spec` local variable (see L98) but the tool paired dissimilar lines. L110-111 is the updated version of L82-83. The diff matched lines L82-L83 with a `+dramsim3` dependency, which is new in this PR.

Reply:
Ah ok should this add `conflicts(~hybridsim, when=~dramsim2)`?

Reply:
Added conflicts for hybridsim